solver/src/driver.rs

```diff
@@ -285,23 +288,78 @@ impl Driver {
         }
     }
 
-    // Record metric with the amount of orders that were matched but not settled in this runloop (effectively queued for the next one)
-    // Should help us to identify how much we can save by parallelizing execution.
-    fn report_matched_but_unsettled_orders(
+    // Record metric with surplus achieved in winning settlement
+    // vs that which was unrealized in other feasible solutions.
+    fn surplus_report(
         &self,
```

Comment:
Since this function doesn't use anything from `self` except `self.metrics` maybe we could refactor it into a pure free standing function that can be independently tested and live in another module so that the driver.rs file doesn't get even bigger. That function could return the count and then here in driver.rs we call the refactored function and set the metric.

Reply:
Sure thing. I will give this a try.

Reply:
Cool, I will surely put it on my list of TODOs though, because it seems fun nonetheless.